.github/workflows/ci.yml

```diff
@@ -87,6 +87,23 @@ jobs:
       run: |
         pytest -v tests
 
+  test-pyinstaller-build:
+    name: Test pyinstaller
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v3
+    - name: Set up Python 3.9
+      uses: actions/setup-python@v4
+      with:
+        python-version: 3.9
+    - name: Install package and dev dependencies
+      run: |
+        python -m pip install --upgrade pip
```

Comment:
Is the current pip outdated?

Reply:
I just copy pasted this, will remove it

Reply:
Actually, no, I'll keep it around. The other jobs use the command as well.